src/App.tsx

```diff
@@ -1,24 +1,180 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import {
+  addTodos,
+  deleteTodos,
+  editTodos,
+  getTodos,
+} from './api/todos';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Notification } from './components/Notification';
+import { TodoItem } from './components/TodoItem';
+import { Errors } from './types/Errors';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 6398;
+
+const filterTodos = (todos:Todo[], selectedStatus: TodoStatus) => (
+  todos.filter(todo => {
+    switch (selectedStatus) {
+      case TodoStatus.Active:
+        return !todo.completed;
+
+      case TodoStatus.Completed:
+        return todo.completed;
+
+      case TodoStatus.All:
+      default:
+        return true;
+    }
+  })
+);
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedStatus, setSelectedStatus] = useState(TodoStatus.All);
+  const [errorType, setErrorType] = useState(Errors.None);
+  const [hasErrorNotification, setHasErrorNotification] = useState(false);
+  const [updatingTodoIds, setUpdatingTodoIds] = useState<number[]>([]);
+
+  const hideErrorNotifications = () => setHasErrorNotification(false);
+  const showErrorNotification = (error:Errors) => {
+    setErrorType(error);
+    setHasErrorNotification(true);
+    setTimeout(hideErrorNotifications, 3000);
+  };
+
+  const fetchAllTodos = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      showErrorNotification(Errors.Get);
+    }
+  };
+
+  useEffect(() => {
+    fetchAllTodos();
+  }, []);
+
+  const addNewTodo = async (title: string) => {
+    if (title.trim()) {
+      try {
+        const newTodo = {
+          title,
+          userId: USER_ID,
+          completed: false,
+        };
+
+        await addTodos(newTodo);
```

Comment:
check in previous task description how should work the process of adding new todos, you didn't complete all requirements (check points with temp todo)